charts/rancher-cis-benchmark/0.2.1/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.0-alpha1
```

Comment:
Should be 2.6.1, and 0.2.1 should get max of 2.6.0. Do that in this last commit.